aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/pom.xml

```diff
@@ -15,6 +15,18 @@
   <version>1.0.0</version>
   <packaging>jar</packaging>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>spring-cloud-azure-dependencies</artifactId>
+        <version>4.0.0-beta.3</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
```

Comment:
@backwind1233 should this dependencyManagement be in the parent pom file or in each sample's pom?

Reply:
Parent pom already have dependency management.
We should delete the `dependencyManagement` here after 4.0.0-beta.3(or 4.0.0) released.

Reply:
How to make sure that will be done?

Reply:
Issue created: https://github.com/Azure/azure-sdk-for-java/issues/25828